examples/threadExample/src/main/java/io/realm/examples/threads/ThreadExampleActivity.java

```diff
@@ -84,10 +82,13 @@ public SectionsPagerAdapter(FragmentManager fm) {
 
         @Override
         public Fragment getItem(int position) {
-            switch(position) {
-                case 0: return new ThreadFragment();
```

Comment:
It's a bit dangerous in switch-case since it makes it easier to forget to break if needed. Anyway, I don't feel too strongly about it.